gatsby-config.js

```diff
@@ -117,6 +117,16 @@ module.exports = {
             },
           },
         },
+        tessen: {
+          product: 'DOC',
+          subproduct: 'TDOC',
+          segmentWriteKey: 'AEfP8c1VSuFxhMdk3jYFQrYQV9sHbUXx',
+          trackPageViews: true,
+          pageView: {
```

Comment:
i can't quite recall, but we explicitly send the URL path along with the Tessen.page() call

Reply:
I do see it there in the screenshot above https://github.com/newrelic/docs-website/pull/875#discussion_r577186771

Definitely something we should add as well

Reply:
I believe I've covered this in https://github.com/newrelic/docs-website/pull/875/commits/4df3d72dfcd5a8b13f34e4c7e920cbd191d5750c